homeassistant/components/samsungtv/bridge.py

```diff
@@ -358,6 +358,14 @@ def _get_remote(self, avoid_open: bool = False) -> Remote:
                 self._notify_callback()
             except (WebSocketException, OSError):
                 self._remote = None
+            else:
+                if self.token != self._remote.token:
+                    LOGGER.debug(
+                        "SamsungTVWSBridge has provided a new token %s",
+                        self._remote.token,
+                    )
+                    self.token = self._remote.token
+                    # TODO: copy the new token into the config entry
```

Comment:
If we don't update the config entry, then it we cannot use it again after a restart.